internal/pkg/aws/sessions/sessions.go

```diff
@@ -155,6 +155,11 @@ func (p *Provider) defaultSession() (*session.Session, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if _, credErr := validateCredentials(sess); credErr != nil {
```

Comment:
can we also include this for the `Profile` function 🥺

Reply:
Done